server/src/middleware/auth.js

```diff
@@ -0,0 +1,21 @@
+import jwt from "jsonwebtoken";
+
+export const verifyToken = async (req, res, next) => {
```

Comment:
- lets export it at the end of the page just to stay consistent with other parts of the project.